src/scss/_variables.scss

```diff
@@ -24,11 +24,10 @@ $docs-purple: #5300cc;
 $emerald: #2aac02;
 $base-grey: #2e3c43;
 $light-grey: rgba($base-grey, 0.02);
-$typo0: #2e3c43;
```

Comment:
You can't remove it directly as it's used in some variable (in hangar is used in the footer.scss file for the hover). Maybe is better to wait a bit before doing it, adding a deprecated comment or similar to be sure that we will not use it anymore

Reply:
oki 👍🏼